tc_prometheus/metrics/prometheus_metrics.py

```diff
@@ -19,7 +20,9 @@ def __init__(self, config):
             port = config.PROMETHEUS_SCRAPE_PORT
             if isinstance(port, str):
                 port = int(port)
-            start_http_server(port)
+            registry = CollectorRegistry()
+            multiprocess.MultiProcessCollector(registry)
+            start_http_server(port, registry=registry)
```

Comment:
```suggestion
            if environ.get('PROMETHEUS_MULTIPROC_DIR') is not None:
                registry = CollectorRegistry()
                multiprocess.MultiProcessCollector(registry)
                start_http_server(port, registry=registry)
            else:
                start_http_server(port)
```

Reply:
As this is open for very long (and I am sorry for that), I will wait for a week or two, to wait for a reaction. In case the PullRequest owner is not reacting, I will probably create my own PullRequest instead, but I would like to merge this one to not lose history and not lose the owner of the change in the git history.